spec/unit/lib/cloud_controller/diego/buildpack/staging_action_builder_spec.rb

```diff
@@ -45,13 +45,14 @@ module Buildpack
             app_bits_checksum: { type: 'sha256', value: 'package-checksum' }
           }
         end
+        let(:legacy_md5_buildpack_paths_enabled) { false }
         let(:buildpacks) { [] }
         let(:generated_environment) { [::Diego::Bbs::Models::EnvironmentVariable.new(name: 'generated-environment', value: 'generated-value')] }
 
         before do
           allow(LifecycleBundleUriGenerator).to receive(:uri).with('the-buildpack-bundle').and_return('generated-uri')
           allow(BbsEnvironmentBuilder).to receive(:build).with(env).and_return(generated_environment)
-          TestConfig.override(credhub_api: nil)
+          TestConfig.override(staging: { legacy_md5_buildpack_paths_enabled: }, credhub_api: nil)
```

Comment:
Other config options are defined above (`let(:config)`) and passed to the constructor. `legacy_md5_buildpack_paths_enabled` should also be added there.